<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>534</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>497</width>
    <height>534</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>497</width>
    <height>534</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Tn Symmetry Statistic</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="topMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="parentLayout">
      <item>
       <layout class="QVBoxLayout" name="mainLayout">
        <item>
         <widget class="QLabel" name="instructionsLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="lineWidth">
           <number>1</number>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/img/logo&quot;/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Instructions:&lt;/span&gt; Load file containing data in .dat or .csv format. Select a symmetry point to perform the test.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="textFormat">
           <enum>Qt::RichText</enum>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="fileLayout">
          <item>
           <widget class="QLineEdit" name="fileEdit">
            <property name="inputMask">
             <string/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>Path to file</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="fileButton">
            <property name="toolTip">
             <string>Load file in .dat format (data aligned in columns) or .csv format (data separated by commas).</string>
            </property>
            <property name="text">
             <string>Load file</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QFormLayout" name="valuesLayout">
          <item row="0" column="0">
           <widget class="QGroupBox" name="singlevalueBox">
            <property name="toolTip">
             <string>Test symmetry using a single value of c.</string>
            </property>
            <property name="title">
             <string>Single value</string>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <layout class="QFormLayout" name="formLayout_2">
             <item row="1" column="0">
              <layout class="QHBoxLayout" name="cLayout">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>c:</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="cvalueEdit">
                 <property name="toolTip">
                  <string>Point of simmetry.</string>
                 </property>
                 <property name="text">
                  <string>0.000</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QGroupBox" name="multiplevaluesBox">
            <property name="mouseTracking">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Compute the value of Tn for a given interval.</string>
            </property>
            <property name="title">
             <string>Multiple values</string>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <layout class="QFormLayout" name="formLayout">
             <item row="0" column="0">
              <layout class="QVBoxLayout" name="multcBox">
               <item>
                <layout class="QHBoxLayout" name="minclayout">
                 <item>
                  <widget class="QLabel" name="mincLabel">
                   <property name="text">
                    <string>Minimum c:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="minimumcEdit">
                   <property name="toolTip">
                    <string>Minimum value of symmetry to test.</string>
                   </property>
                   <property name="text">
                    <string>-0.001</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="maxcLayout">
                 <item>
                  <widget class="QLabel" name="maxcLabel">
                   <property name="text">
                    <string>Maximum c:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="maximumcEdit">
                   <property name="toolTip">
                    <string>Maximum value of symmetry to test.</string>
                   </property>
                   <property name="text">
                    <string>0.001</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="deltacLayout">
                 <item>
                  <widget class="QLabel" name="deltacLabel">
                   <property name="text">
                    <string>Delta c:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="deltacEdit">
                   <property name="toolTip">
                    <string>Step interval for the symmetry values.</string>
                   </property>
                   <property name="text">
                    <string>0.00001</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="plotsaveLayout">
        <item>
         <widget class="QPushButton" name="resultsButton">
          <property name="text">
           <string>Get Results</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="saveButton">
          <property name="text">
           <string>Save to file</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>497</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>fileEdit</tabstop>
  <tabstop>cvalueEdit</tabstop>
  <tabstop>minimumcEdit</tabstop>
  <tabstop>maximumcEdit</tabstop>
  <tabstop>deltacEdit</tabstop>
  <tabstop>resultsButton</tabstop>
  <tabstop>fileButton</tabstop>
  <tabstop>singlevalueBox</tabstop>
  <tabstop>multiplevaluesBox</tabstop>
  <tabstop>saveButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
